packages/ui/src/examples/E1_FetchWithAxios.tsx

```diff
@@ -2,24 +2,45 @@
 import { useEffect, useState } from 'react'
 
 import { Box } from '@chakra-ui/react'
+import axios from 'axios'
 import { useRouter } from 'next/router'
 
 import { API_URL } from '@wsvvrijheid/config'
 
 // https://wsvv-api-staging.onrender.com/api/blogs?locale=tr
 // You can use local API_URL instead of the above url
 // yarn --cwd apps/api dev to run the api locally (http://localhost:1337)
-const BLOG_URL = `${API_URL}/api/blogs`
+const BLOG_URL = "https://wsvv-api-staging.onrender.com/api/blogs?locale=tr"
 
 export const FetcWithAxios = () => {
   const [blogs, setBlogs] = useState([])
```

Comment:
```suggestion
  const [blogs, setBlogs] = useState<Blog[]>([])
```

You can specify the type of the state. It helps.